core/templates/App.ts

```diff
@@ -71,6 +66,11 @@ require('google-analytics.initializer.ts');
 // loaded after app.constants.ts
 require('I18nFooter.ts');
 
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
```

Comment:
I thought this PR was supposed to remove this?

Reply:
This PR is not meant to remove UpgradedService from this file. It just removes the unnecessary services.